opentech/apply/funds/views.py

```diff
@@ -197,6 +190,18 @@ def dispatch(self, request, *args, **kwargs):
             raise PermissionDenied
         return super().dispatch(request, *args, **kwargs)
 
+    @property
+    def transitions(self):
+        transitions = self.object.get_available_user_status_transitions(self.request.user)
+        return {
+            transition.name: transition
+            for transition in transitions
+        }
+
+    def buttons(self):
+        yield ('save', 'Save')
+        yield from ((transition, transition.title) for transition in self.transitions)
```

Comment:
this yields `Save | <Lambda> | Submit` for me

Reply:
Could not reproduce. What state and what application?

Reply:
46, Under discussion (post internal review) with more info required. From dev with a reset phase post fsm merge.